test/conftest.py

```diff
@@ -171,6 +171,18 @@ def pudl_out_eia(live_dbs, pudl_engine, request):
     )
 
 
+@pytest.fixture(scope="session", name="fast_out_annual")
+def fast_out_annual(pudl_engine, pudl_datastore_fixture):
+    """A PUDL output object for use in CI."""
+    return pudl.output.pudltabl.PudlTabl(
+        pudl_engine,
+        freq="AS",
+        fill_fuel_cost=True,
+        roll_fuel_cost=True,
+        fill_net_gen=True,
+    )
+
```

Comment:
i think moving `fast_out_annual` here is a good idea but i think you should also remove it from `output_test.py` this way the ppe/ferc-eia won't need to be regenerated twice.